bulbs/content/tasks.py

```diff
@@ -34,3 +33,21 @@ def index_feature_type_content(featuretype_pk):
     featuretype = FeatureType.objects.get(pk=featuretype_pk)
     for content in featuretype.content_set.all():
         content.index()
+
+
+@shared_task(default_retry_delay=5)
+def update_feature_type_rates(featuretype_pk):
+    from bulbs.contributions.models import ContributorRole, FeatureTypeRate, FEATURETYPE
```

Comment:
Is this non-standard to put import statements in method declarations like this? // @benghaziboy @MichaelButkovic